src/commands/HotpepperSearch.ts

```diff
@@ -13,21 +14,27 @@ export function searchHotpepper(c: vscode.ExtensionContext): { dispose: any } {
     const generalConf: GeneralConfig = generalConfigImpl.getGeneralConf();
 
     // parameter 作成
-    const params: HotpepperApiForm = {
-      // lat: generalConf.latLng.lat,
-      // lng: generalConf.latLng.lng,
-      lat: 35.6198513,
-      lng: 139.7281892,
-    };
-    console.log(params);
+    const priceRange = generalConf.priceRange?.sort();
+    const priceMin = priceRange === undefined ? 0 : priceRange[0];
+    const priceMax = priceRange === undefined ? 0 : priceRange[1];
```

Comment:
```suggestion
    const priceMax = priceRange === undefined ? priceMin : priceRange[1];
```
こうしないと、priceMin ≦ priceMaxの関係が保たれない